lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -1,18 +1,78 @@
 # -*- coding: utf-8 -*-
-
 # Copyright: Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
 
-# NOTE: this file is here to allow modules using the new attributes feature to
-#       work w/o errors in this version of ansible, it does NOT provide the full
-#       attributes feature, just a shim to avoid the fragment not being found.
-
 class ModuleDocFragment(object):
 
     # Standard documentation fragment
     DOCUMENTATION = r'''
 options: {}
+attributes:
+    check_mode:
```

Comment:
Is it possible that we could just do the same as `options`, and make these an empty dict for backwards compat?  Technically this is a 2.12 feature, so I don't see why we need populated `attributes` on older versions.  I think we should just make the doc fragment discoverable, and leave it empty.

Reply:
dont even need attributes at all, just thought it would be nice to display, i can take em out if you wish

Reply:
I would prefer they be removed if not needed.